go/interpreter/lfvm/instructions_test.go

```diff
@@ -218,10 +218,11 @@ func TestCallChecksBalances(t *testing.T) {
 		params: tosca.Parameters{
 			Recipient: source,
 		},
-		context: runContext,
-		stack:   NewStack(),
-		memory:  NewMemory(),
-		gas:     1 << 20,
+		context:  runContext,
+		stack:    NewStack(),
+		memory:   NewMemory(),
+		gas:      1 << 20,
+		revision: tosca.R09_Berlin,
 	}
```

Comment:
- add tests where account does not exist.
- add test where address is not in access list.